lib/ui/hooks.dart

```diff
@@ -30,9 +84,98 @@ void _updateDisplays(
   PlatformDispatcher.instance._updateDisplays(displays);
 }
 
+List<DisplayFeature> _decodeDisplayFeatures({
```

Comment:
Both `_decodeDisplayFeatures` and `_buildViewConfiguration` were moved from `PlatformDispatcher`.

Reply:
Is this a strictly necessary change? If this is a refactoring, I'd recommend creating a separate follow-up pull request as this pull request is a bit large already

Reply:
Yes, this is necessary, because this part of logic is now shared between `_updateWindowMetrics` and `_addView` to parse the window metrics packet. I'd be happy to submit a preliminary PR for this.

Reply:
Ah gotcha makes sense. Yup a preliminary PR would be great 👍